src/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/Pkcs12Builder.cs

```diff
@@ -0,0 +1,354 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Security.Cryptography.Asn1;
+using System.Security.Cryptography.Pkcs.Asn1;
+using Internal.Cryptography;
+
+namespace System.Security.Cryptography.Pkcs
+{
+    public sealed class Pkcs12Builder
+    {
+        private ReadOnlyMemory<byte> _sealedData;
+        private List<ContentInfoAsn> _contents;
+
+        public bool IsSealed => !_sealedData.IsEmpty;
+
+        public void AddSafeContentsEncrypted(
+            Pkcs12SafeContents safeContents,
+            byte[] passwordBytes,
+            PbeParameters pbeParameters)
+        {
+            AddSafeContentsEncrypted(
+                safeContents,
+                // Allows null.
```

Comment:
What does this comment mean?  Just that the ROS ctor is tolerant of null arrays, or something else?

Reply:
Yeah, it made more sense when it was an extension method invocation.